README.md

```diff
@@ -1,41 +1,45 @@
+
 ![logo](https://i.imgur.com/2nu3hX1.png)
 
+WebScanner project - tool for monitoring servers, websites and services.
+
+#  Goal
+
+Main goal is to provide server utility which can send requests in order specified by implemented Scheduler. Specific responses will be saved in database operated by WebScanner. 
+
+#  Language
 
-WebScanner project -  tool for monitoring servers, websites and services
-# Goal
-General goal is to provide universal server tool being able to send requests in order specified by implemented Scheduler. Specific responses will be saved in database operated by WebScanner. Configuration will be possible by using build-in application interface.
-# IDE
-VS Code + plugins OR Visual Studio + Resharper
-# Language
 C# 7.0
-# Platform
+
+#  Platform
+
 .NET Core 2.1
-# Scope(still open)
+
+#  Scope(still open)
+
 * Repository structure in a manner of S. Allen's approach
 * C# naming conventions
 * Using c# design patterns
 * Usage of FluentAssertions
 * DDD
 * Reference materials (e.g. books, articles, lectures)
 * Readme-driven development
-# Reason for creating the application
-Application is aimed to people who want to manage their network resorces by sending specific requests
-to endpoints. Application is also to be used by developers, we try to build app in DDD aproach, so it will be easy to maintain and expand. All people who want to work with us or personalize
+
+#  Reason for creating the application
+
+Application is aimed at people who want to manage their network resources by sending specific requeststo endpoints. Application is also can be used by developers, we try to build app in DDD aproach, so it will be easy to maintain and expand. All people who want to work with us or personalize
+
 WebScanner are welcome to do so! Feel free to contact us!
-# Technologies
+
+#  Technologies
+
 * Quartz.NET for scheduling tasks
 * PostgreSQL for saving configuration and responses
 * RESTful api for serving endpoints for configuring application
-# Application interface
-* Adding order: POST [ip]/order/add
-
- ARGUMENT: Order order ( MUST ADD A MODEL STRUCTURE )
-* Deleting order: POST [ip]/order/delete
-
- ARGUMENT: int orderId
- 
- # Solution Structure
- coming soon!
- # Project schedule
- coming soon!
```

Comment:
I don't see any point in this. We use redmine to project management and schedule.